inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/yaml/KnowledgeBaseMapping.java

```diff
@@ -74,7 +79,8 @@
         this.propertyTypeIri = vf.createIRI(propertyTypeIri);
         this.propertyLabelIri = vf.createIRI(propertyLabelIri);
         this.propertyDescriptionIri = vf.createIRI(propertyDescriptionIri);
-    }
+        this.rootConcepts = rootConcepts;
```

Comment:
Method argument names should start with `a` (e.g. `aRootConcepts`)  and then the `this.` is not needed.